JuiceMaker/JuiceMaker/Controller/FixStockViewController.swift

```diff
@@ -1,30 +1,119 @@
+        guard let fruit = fruitStore?.banana else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        bananaLabel.text = fruitStore?.banana.stock.description
     }
-    @IBAction func starawberryStepper(_ sender: UIStepper) {
+    @IBAction func pineappleStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.pineapple else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        pineappleLabel.text = fruitStore?.pineapple.stock.description
     }
-    @IBAction func bananaStepper(_ sender: UIStepper) {
+    @IBAction func kiwiStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.kiwi else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        kiwiLabel.text = fruitStore?.kiwi.stock.description
     }
-    @IBAction func pineappleStepper(_ sender: UIStepper) {
+    @IBAction func mangoStepperValueChanged(_ sender: UIStepper) {
+        guard let fruit = fruitStore?.mango else {
+            return
+        }
+        tryStepper(fruit: fruit, sender: sender)
+        mangoLabel.text = fruitStore?.mango.stock.description
     }
-    @IBAction func kiwiStepper(_ sender: UIStepper) {
+    
+    private func tryStepper(fruit: Fruit, sender: UIStepper) {
+        do {
+            if sender.value < 0 {
+                try fruitStore?.decreaseStock(fruit: fruit)
+            } else {
+                try fruitStore?.increaseStock(fruit: fruit)
+            }
+            sender.value = 0
+        } catch {
+            sender.value = 0
+        }
     }
-    @IBAction func mangoStepper(_ sender: UIStepper) {
+    
+    @IBAction func closeButton(_ sender: UIBarButtonItem) {
+        delegate?.fixStockViewControllerDidTapCloseButton(self)
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func showAlert(message: JuiceMakerError){
+        let alert = UIAlertController(title: nil, message: message.description, preferredStyle: .alert)
+        let cancelAction = UIAlertAction(title: "예", style: .cancel)
+        
+        alert.addAction(cancelAction)
+        present(alert, animated: true, completion: nil)
     }
+    
+    func typeChange(of stockInString: String?) throws -> Int {
+        guard let stock = stockInString, let stockInInt = Int(stock) else {
+            throw JuiceMakerError.unexpectedError
+        }
+        return stockInInt
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        strawberryLabel.text = String(fruitStore?.strawberry.stock ?? 0)
+        bananaLabel.text = String(fruitStore?.banana.stock ?? 0)
+        pineappleLabel.text = String(fruitStore?.pineapple.stock ?? 0)
+        kiwiLabel.text = String(fruitStore?.kiwi.stock ?? 0)
+        mangoLabel.text = String(fruitStore?.mango.stock ?? 0)
+        
+        setUpStepper(stepper: strawberryStepper)
```

Comment:
[Style]
5줄을 2줄로 줄일 수 있다?! ㅋㅋㅋ
이런식으로도 표현할 수 있수도 있을 것 같아요 (스타일 적인거라 필수는 아니고 참고용으로만 봐주세요)

```suggestion
      let steppers = [bananaStepper, kiwiStepper, ....]
      steppers.forEach { setUpStepper(stepper: $0) }
```